### PR TITLE
change time format to ISO format in samples cache

### PR DESCRIPTION
**Pull request by annyhe**

### CODE REVIEW

File: tests/cache/models/samples/get.js [hunk 1]
```diff
@@ -39,6 +39,24 @@ describe(`api::redisEnabled::GET ${path}`, () => {
   after(rtu.forceDelete);
   after(() => tu.toggleOverride('enableRedisSampleStore', false));
 
+  it('time-realted fields have the expected format', (done) => {
```
Comment (iamigo) [inline tests/cache/models/samples/get.js]: related

**Comment by annyhe** [comment]
somehow I cannot login locally, so will push to staging to test whether the timeout works.

**Comment by annyhe** [comment]
Timeout works on staging. Here's how to reproduce:
PATCH aspect http://refocus-staging.internal.salesforce.com/v1/aspects/AVGPAGETIME
with 
{"timeout":"10s"}
POST to http://refocus-staging.internal.salesforce.com/v1/samples/upsert/bulk
with 
[{"name":"Salesforce.SFDC_Core.CHI.SP3.CS24|QPID","value":"3"},
{"name":"Salesforce.SFDC_Core.CHI.SP3.CS27|AVGPAGETIME","value":"1"}, 
{"name":"Salesforce.SFDC_Core.CHI.SP3.NA19|AVGPAGETIME","value":"1"},
{"name":"Salesforce.SFDC_Core.CHI.SP3.NA20|AVGPAGETIME","value":"1"}]

the specified samples should timeout after 10s

**Comment by iamigo** [comment]
And you tried this with the redis sampleStore feature both enabled and **not** enabled?

**Comment by annyhe** [comment]
@iamigo yes I tried with redis sampleStore feature both enabled and not enabled

**Pull request merged**